installing/install_config/customizations.adoc

```diff
@@ -29,46 +29,55 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configure the Kubernetes API server.
```

Comment:
@adellape, do you have information about configuring the API server?

Reply:
need a section on configuring the openshift-api server also.

and the kuberenetes controller and the openshift controller.
@openshift/sig-master and @openshift/sig-developer-experience can help w/ those.

Reply:
The OpenShift-api server "plugs into" the kube-apiserver, there is no separate configuration exposed.

Reply:
@sanchezl it has its own operator, therefore it has its own operator resource.  At a minmum this means independent config for things like loglevel (both loglevel of the operator and loglevel of the operand).

Reply:
@bparees I didn't notice that this file had split, there was a whole other section on all the custom resource definitions, and the "missing" resources were listed there. This table is only about the "Global Configuration".  Unfortunately, the resource type for the global configuration is "clusterperators".  I would expect the _actual_ "Cluster Operator" resources should be in another section (similar to how they are split up in the console UI).

Reply:
Because of the lack of information about the individual CRDs, I commented out the table. If you're willing to get more information about each, we can bring it back as an RFE.

Reply:
>  If you're willing to get more information about each

that is part of the information teams should be working with their docs team members to build, per my recent email about the need to document this stuff.